base/index.mustache

```diff
@@ -10,23 +10,23 @@ app.use(bodyParser.json());
 app.use(cors());
 
 fs.readdirSync('./middleware').forEach(name => {
-  app.use(require('./middleware/' + name));
+  app.use(require(`./middleware/${name}`));
 });
 
 fs.readdirSync('./routes').forEach(name => {
-  const route = require('./routes/'+name);
+  const route = require(`./routes/${name}`);
   app.use(route.http);
   route.ws(io);
 });
 
-app.use(function (err, req, res, next) {
+app.use((err, req, res) => {
```

Comment:
Is this to pass a linter rule or just because? If our linter rules mandate that we always use fatarrows and never traditional functions that might be a problem. Doesn't matter here, but there will probably be times when we want the old version of `this` binding. Especially when using older libs.